docs/gx_time_frequency.md

```diff
@@ -4,22 +4,27 @@ title: Time-Frequency component
 tagline: GreenX Time-Frequency
 description: Time-Frequency component
 ---
+# General
+
+The time-frequency component of GreenX provides minimax time and frequency grids and the corresponding quadrature weights for the numerical evaluation of time and frequency integrals as well as the weights for Fourier transforms between time and frequency grids. The minimax grids are primarily relevant for low-scaling RPA and GW algorithms, which use the [space-time method](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.74.1827). However, the compact minimax frequency grids also reduce the computational prefactor in RPA codes with conventional scaling. In addition, the time grids can be employed in Laplace-transformed direct MP2 (LT-dMP2) calculations.
 
 # Benchmarks
 
-## CH<sub>4</sub> RPA
+## Correlation energy from conventional RPA for CH<sub>4</sub>
 
-In this test, we evaluate the RPA total energy of CH4 using a Gauss-Legendre grid, a modified Gauss-Legendre grid (so 
-far the standard in FHI-aims and abinit), and minimax grids. An accuracy of 10^-6 eV is reached with 10 minimax grid 
-points while the modified Gauss-Legendre grids requires 36 points for this accuracy.
+In this test, we evaluate the RPA total energy of CH4 using a Gauss-Legendre grid, a modified Gauss-Legendre grid (so far the standard in FHI-aims and abinit), and minimax grids. An accuracy of 10^-6 eV is reached with 10 minimax grid points while the modified Gauss-Legendre grids requires 36 points for this accuracy.
 
 [CH4 benchmark](./img/ch4_bench.png)
```

Comment:
Here the code for images is wrong, I forgot to add the ! symbol:

![CH4 benchmark](./img/ch4_bench.png)

Otherwise we only see a link instead of the image

Reply:
corrected the figure